Gemfile.lock

```diff
@@ -100,6 +100,12 @@ GEM
       tzinfo (~> 2.0)
     addressable (2.8.0)
       public_suffix (>= 2.0.2, < 5.0)
+    adminterface (0.3.0)
+      activeadmin (~> 2.9)
```

Comment:
This is a problem. We need ActiveAdmin 2.11 for Rails 7 and the fact this repo is unmaintained now (not accepting pull requests either) means we'd definitely have to fork this whole project to work around this.